### ISSUE TITLE
Unexpected values returned from streamActionTraces() when with_dbops === true

### ISSUE THREAD

**Issue by wlegate** [dfuse-io/client-js#4]
**Summary**
The data returned from the streamActionTraces() method doesn't match the format expected by the declared Typescript interfaces.

**Steps to Reproduce**
Call the streamActionTraces() method with the `with_dbops` option set to true.

**Expected Results**
When an inbound message is received of type InboundMessageType.ACTION_TRACE, the message should conform to the ActionTraceInboundMessage interface declaration.

**Actual Results**
The `dbOps` key is undefined. The message contains a `dbops` key instead which is an array of objects similar to the following:

```
{
  "op": "UPD",
  "action_idx": 3,
  "opayer": "sampleuser24",
  "npayer": "sampleuser24",
  "path": "mycontract11//mytable12345/",
  "old": "000000000000000050c8102afb43affa97f89a5c881300000000000004454f53000000006d3301000000000001",
  "new": "000000000000000050c8102afb43affa97f89a5c881300000000000004454f53000000004de500000000000001"
}
```

…which is different than the DbOp type declaration in the @dfuse/client library:

```
export declare type DbOp<T> = {
    op?: string;
    action_idx: number;
    account?: string;
    table?: string;
    scope?: string;
    key?: string;
    old?: DbRow<T>;
    new?: DbRow<T>;
};
```

**Comment by maoueh** [comment]
@wlegate Thanks for reporting, it was indeed completely wrong. I fixed a big bunch of problems related to `*ops` typings at the same time.

I published `0.2.4` a few minutes ago that should fix all that. Don't hesitate to report back if you still face any typings issues.

**Comment by wlegate** [comment]
Awesome, thank you so much for the quick response & fix @maoueh!
